CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 3.0.0 (February 24, 2020)
+
+BREAKING CHANGES
+* ns1_zone: Adds fully defined suboptions for `secondary` and `primary` options. These previously accepted arbitrary dict as values.
+* ns1_zone: Returned zone data is now returned in a key named `zone` (was previously returned in `data`).
+
+ENHANCEMENTS
```

Comment:
not a big deal really, but I think enhancements and improvements are essentially the same thing

Reply:
my b, meant to list everything in `enhancements` in a `features` section